test/verify/check-kdump

```diff
@@ -68,7 +70,8 @@ class TestKdump(KdumpHelpers):
         m = self.machine
 
         b.wait_timeout(120)
-        m.execute("systemctl enable kdump; systemctl start kdump || true")
+        if m.image.startswith("fedora"):
+            m.execute("systemctl enable kdump; systemctl start kdump || true")
```

Comment:
Can we lose the `|| true` here? I hate these, they just make tests unpredictable later on. Also, `enable --now`